packages/wrangler/src/api/pages/create-worker-bundle-contents.ts

```diff
@@ -0,0 +1,64 @@
+import { readFileSync } from "node:fs";
+import path from "node:path";
+import { Response } from "undici";
+import { createWorkerUploadForm } from "../../create-worker-upload-form";
+import type { BundleResult } from "../../bundle";
+import type { CfWorkerInit } from "../../worker";
+import type { Blob } from "node:buffer";
+import type { FormData } from "undici";
+
+/**
+ * Takes a Worker bundle - `BundleResult` - and generates the contents
+ * of the upload worker bundle file
+ */
+export async function createUploadWorkerBundleContents(
```

Comment:
personally I think is one level of abstraction too many. `createWorkerBundleFormData` should prolly be enough. Whoever consumes this API can decide what they want to do with the formData, but maybe that's me, so I'm asking for a second opinion here :)

`createUploadWorkerBundleContents` is called twice in `publish`

Reply:
I think this is fine as you've got it :)

Reply:
yeah, you are right. this came very in handy for the `functions build` work, so I take my comment back 😅